packages/sdk-components-animation/src/stagger-animation.tsx

```diff
@@ -0,0 +1,43 @@
+import { forwardRef, type ElementRef } from "react";
+
+const easings = {
```

Comment:
Why not array?

Reply:
It's public API in private API it's key -> function

Reply:
So this is same in public and private
easing?: keyof typeof easings;